src/components/atoms/NavLink/NavLink.tsx

```diff
@@ -1,16 +1,20 @@
 import { NavLink as Link } from '@mantine/core';
+import { Link as TanstackLink } from '@tanstack/react-location';
 import type { ReactNode } from 'react';
 import { useState } from 'react';
 
 interface NavLinkProps {
   label: string;
   icon: ReactNode;
+  path: string;
 }
 
 export const NavLink = ({ links }: { links: NavLinkProps[] }) => {
-  const [isActive, setIsActive] = useState(0);
+  const [isActive, setIsActive] = useState(-1);
```

Comment:
I don't understand why don't we use booleans?
if there is more than two cases, it's better to use some literal strings, not numbers

Reply:
ok let's just leave this be for now